neo/Consensus/ConsensusService.cs

```diff
@@ -240,20 +246,6 @@ private void OnPrepareRequestReceived(ConsensusPayload payload, PrepareRequest m
             context.SignedPayloads[payload.ValidatorIndex] = message.PrepReqSignature;
             context.FinalSignatures[payload.ValidatorIndex] = message.FinalSignature;
 
-            if (!CheckPrimaryPayloadSignature(payload))
-            {
-                context.SignedPayloads[payload.ValidatorIndex] = null;
-                return;
-            }
-
```

Comment:
This was removed because in this current code we check all partial signatures upon receiving them (it is now possible because we always known the PrepareReqPayload associated to it)